packages/twenty-server/src/utils/chunk-array.ts

```diff
@@ -0,0 +1,11 @@
+export const chunkArray = <T>(array: T[], chunkSize = 5): T[][] => {
```

Comment:
feels a bit like re-inventing the wheel here. https://www.geeksforgeeks.org/lodash-_-chunk-method/
Non blocking